static/schemas/2.0/skus.2.0.json

```diff
@@ -0,0 +1,248 @@
+{
+    "title": "Skus.2.0",
+    "definitions": {
+        "SkusRules": {
+            "description": "Possible Rules a Sku can use",
+            "type": "array",
+            "items": {
+                "type": "object",
```

Comment:
The rule object isn't correct, firstly there should be a property for required `path`, but you forgot it; Then the rule words can only pick one of it or no rule word, the correct example should be:
```
"items": {
    "type": "object",
    "required": ["path"],
    "oneOf": [
        {
            "maxProperties": 1,
            "minProperties": 1
        },
        {
            "maxProperties": 2,
            "minProperties": 2
        }
    ],
    "additionalProperties": false,
    "properties": {
        "path": {
            "type": "string",
            "minLength": 1
        },
        "contains": {
            "type": "string"
        },
        "greaterThan": {
            "type": "string"
        },
        "minLength": {
            "type": "string"
        },
        {
            ...
        }
    }
}
```